packages/test/test-end-to-end-tests/src/test/summarizeRestart.spec.ts

```diff
@@ -44,6 +44,10 @@ describeNoCompat("Summarizer closes instead of refreshing", (getTestObjectProvid
 	itExpects(
```

Comment:
In the beforeEach above let's remove the SummaryStateUpdateMethodV2 setting.  Also in packages/test/test-end-to-end-tests/src/test/summarizerWithLocalChanges.spec.ts